flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -214,6 +214,8 @@ public void setup() throws IOException {
                 new BlobCacheService(new Configuration(), new VoidBlobStore(), null);
 
         configuration = new Configuration();
+        configuration.set(TaskManagerOptions.TASK_HEAP_MEMORY, MemorySize.ofMebiBytes(10));
+        configuration.set(TaskManagerOptions.MANAGED_MEMORY_SIZE, MemorySize.ofMebiBytes(10));
```

Comment:
I think the assumption is that memory sizes for all components should be explicitly configured before starting the TM process. Having only these two options set and the test doesn't fail is an indicator that we do not check the assumption strictly.